connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -1303,7 +1303,7 @@ private void reconfigureConnector(final String connName, final Callback<Void> cb
             }
             if (changed) {
                 List<Map<String, String>> rawTaskProps = reverseTransform(connName, configState, taskProps);
-                if (isLeader()) {
+                if (isLeader() || (leaderUrl() != null && leaderUrl().startsWith("NOTUSED"))) {
```

Comment:
Should we start connect rest server if we want this code can running successfully. but I not seeing where to start the connect rest server in MirrorMaker.
In my case, there got an exception which cause for unknown host url